Tests/SwiftDocCTests/Indexing/NavigatorIndexTests.swift

```diff
@@ -395,12 +395,12 @@ Root
         assertUniqueIDs(node: navigatorIndex.navigatorTree.root)
         assertEqualDumps(navigatorIndex.navigatorTree.root.dumpTree(), """
         [Root]
-        ┣╸Objective-C
+        ┣╸occ
```

Comment:
Why this change and can we avoid it? I think this will break clients that use that node's name in UI.

Reply:
Good call- I reworked things to avoid this change.